commands/create/templates/sn/none/test/integration/fixtures/hello.fix.js

```diff
@@ -0,0 +1,17 @@
+export default function fixture() {
+  return {
+    type: 'hello',
+    genericObjects: [
```

Comment:
It would be a nice improvement if it wasn't mandatory to supply a generic object when rendering a viz from a fixture.